docs_user/assemblies/assembly_migrating-ceph-monitoring-stack.adoc

```diff
@@ -0,0 +1,39 @@
+[id="migrating-ceph-monitoring_{context}"]
+
+:context: migrating-ceph-monitoring
+
+= Migrating a {Ceph} cluster with monitoring stack components
+//kgilliga: SMEs, what do you think of the title? ^
+
+In the context of data plane adoption, where the {rhos_prev_long} ({OpenStackShort}) services are
+redeployed in {OpenShift}, a {OpenStackPreviousInstaller}-deployed {CephCluster} cluster will undergo a migration in a process we are calling “externalizing” the {CephCluster} cluster.
+There are two deployment topologies, broadly, that include an “internal” {CephCluster} cluster today: one is where {OpenStackShort} includes dedicated {CephCluster} nodes to host object storage daemons (OSDs), and the other is Hyperconverged Infrastructure (HCI) where Compute nodes
+double up as {CephCluster} nodes. In either scenario, there are some {Ceph} processes that are deployed on {OpenStackShort} Controller nodes: {Ceph} monitors, Ceph Object Gateway (RGW), Rados Block Device (RBD), Ceph Metadata Server (MDS), Ceph Dashboard, and NFS Ganesha.
+The Ceph Dashboard module adds web-based monitoring and administration to the
+Ceph Manager.
+With {OpenStackPreviousInstaller}-deployed {Ceph} this component is enabled as part of the overcloud deploy and it’s composed by:
+
+- Ceph Manager module
+- Grafana
+- Prometheus
+- Alertmanager
+- Node exporter
+
+The Ceph Dashboard containers are included through `tripleo-container-image-prepare` parameters and the high availability relies on `Haproxy` and `Pacemaker` deployed on the {OpenStackShort} front.
+For an external {CephCluster} cluster, High availability is not supported and the work
+is tracked in the https://bugzilla.redhat.com/show_bug.cgi?id=1902212[associated RHCS bugzilla].
+//kgilliga: Do we want the link to this BZ in the downstream docs?
```

Comment:
Yes, it justify the fact that customers are going to lose something since the migration happens. They're not going to have HA anymore, and probably they need to build their own solution for that.

Reply:
I checked with the docs team and they said I should not link to an RFE in downstream or upstream docs. I think we can emphasize that high availability is not supported and leave it at that. 

This is the message I got: "We don’t normally include links to BZs in our docs unless it’s in the release notes, since it’s difficult to maintain. The BZ link might cause customers to hold us responsible for some future work that the devs might not get to within that time frame, etc...."